docker-compose.prod.yml

```
@@ -0,0 +1,100 @@
+  opensearch-data:
+    driver: local
+    driver_opts:
+      type: none
+      o: bind
+      device: /data/opensearch
+  postgresql-data:
+    driver: local
+    driver_opts:
+      type: none
+      o: bind
+      device: /data/postgresql
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)